### PR TITLE
improve getting started

### PR DESCRIPTION
**Pull request by Notalifeform**
make steps to start the server more explicit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,8 +7,13 @@ This is an example implementation of the [@jmondi/oauth2-server](https://github.
 You can use [Foreman](https://github.com/ddollar/foreman) or [Overmind](https://github.com/DarthSim/overmind) to manage these processes. Both tools allow running multiple applications specified in a Procfile simultaneously.
 
 ```
+cp .env.example .env
```
Comment (jasonraimondi) [inline README.md:10]: can you update this to 

```bash
cp --no-clobber .env.example .env
```

thank you

File: README.md [hunk 2]
```diff
@@ -7,8 +7,13 @@ This is an example implementation of the [@jmondi/oauth2-server](https://github.
...
+pnpm install
+cd web && pnpm install && cd ..
```
Comment (jasonraimondi) [inline README.md:12]: Instead of directory traversing here, could we just do

```bash
pnpm install --prefix web
```

**Review by jasonraimondi — CHANGES_REQUESTED** [review]
Thank you for these improvements. Just a few small tweaks on this one to make it a bit easier to grok

File: README.md (hunk 1 above)
Comment (Notalifeform) [inline README.md:10]:
```suggestion
cp -n .env.example .env
```
--no-clobber does not work on osx

File: README.md (hunk 2 above)
Comment (Notalifeform) [inline README.md:12]:
```suggestion
pnpm install --prefix web
```

that looks better indeed

**Review by jasonraimondi — APPROVED** [review]

**Pull request merged**